pkg/virt-config/feature-gates.go

```diff
@@ -28,17 +28,16 @@ const (
 	IgnitionGate      = "ExperimentalIgnitionSupport"
 	LiveMigrationGate = "LiveMigration"
 	// SRIOVLiveMigrationGate enable's Live Migration for VM's with SRIOV interfaces.
-	SRIOVLiveMigrationGate = "SRIOVLiveMigration"
-	CPUNodeDiscoveryGate   = "CPUNodeDiscovery"
-	HypervStrictCheckGate  = "HypervStrictCheck"
-	SidecarGate            = "Sidecar"
-	GPUGate                = "GPU"
-	HostDevicesGate        = "HostDevices"
-	SnapshotGate           = "Snapshot"
-	HotplugVolumesGate     = "HotplugVolumes"
-	HostDiskGate           = "HostDisk"
-	VirtIOFSGate           = "ExperimentalVirtiofsSupport"
-	MacvtapGate            = "Macvtap"
+	SRIOVLiveMigrationGate                = "SRIOVLiveMigration"
+	IgnoreCPUFeaturesDuringSchedulingGate = "IgnoreCPUFeaturesDuringScheduling"
```

Comment:
Feature gates are introduced primarily for two reasons.

1. it's an experimental alpha feature, like snapshots, hotplug ...
2. it's an advanced feature that should only be enabled in specific environments, CPUManager, Sidecar,HostDisk, ....

At some point, feature gates become obsolete and should be deprecated. For instance, a feature that's graduated as being stable should just become enabled by default. This means something like the LiveMigration gate should probably be removed at this point. We haven't defined how to do that yet. 

For the feature gates you're looking at changing here, my expectation is we'd either need to talk about a deprecation policy to remove them, or to leave them as they are.